docs/.vitepress/theme/Styles/main.css

```diff
@@ -30,3 +30,7 @@ pre {
 main a {
 	@apply text-purple-800 underline;
 }
+
```

Comment:
This should happen inside of the markup instead. (https://tailwindcss.com/docs/outline#remove-outlines)

Reply:
Simply add this Tailwind class [here](https://github.com/Bedrock-OSS/bedrock-wiki-vite/blob/wiki/docs/.vitepress/theme/Components/Sidebar/Category.vue#L3)